pvmismatch/contrib/ModuleMismatchSimulator.py

```diff
@@ -0,0 +1,510 @@
+'''
```

Comment:
`#!/usr/bin/env python` seems to be the best practice, but I am having problems testing it on my computer. I have the shebang in the script, and it runs fine when I call python first, but I get an error if I just try to call the script:

```
(pvmismatch_dev) contrib [bennetmeyers] $ ModuleMismatchSimulator.py 
-bash: ModuleMismatchSimulator.py: command not found
```

Am I missing something?